thereseLYR/3D-Model-Ecommerce-Frontend
# chore: setup fe scaffolding

## Changes
- create PR template
- create-react-app folder structure (most of it are defaults, removed tests and reportVitals)

## How to test
- pr template should take effect after this PR is merged
- in 3D-Model-Ecommerce-Frontend folder, run `npm start`, navigate to localhost:3000

## Known Issues
- Indicate known issues, and when they are to be fixed if they are out of scope

## Screenshots (if any)
![image](https://user-images.githubusercontent.com/30279812/190891207-925778ac-322e-477c-83e6-9c04ebf6128f.png)
